napari/components/_tests/test_layer_slicer.py

```diff
@@ -62,6 +63,11 @@ def _get_slice(self, request: FakeSliceRequest) -> FakeSliceResponse:
     def _is_async(self) -> bool:
         return True
 
+    def _slice_dims(self, *args, **kwargs) -> None:
```

Comment:
I added it because the `force_sync` route needs it and I thought the existing methods would remain in the codebase. 

I was struggling with testing and being assured of the routes I was taking. I added the `NotImplementedError` here as a way to check that I made it down that route, though admittedly the name only makes sense in the context of the Fake Layers. 

I can check for an empty dict as you mentioned in another comment and change this to a `pass`.

Reply:
aha, we can also the `slice_count` to check against